failing builds

The builds the recent 0.5.0 PR (#10) worked fine after a rerender, however, the builds on the merged master are now failing. It seems to have something to do with authentication, but I don't know how to resolve it. I don't see anything relevant to this in the conda forge documentation. I tried restarting the travis build but it didn't help.

I think the failures are due to a bug see [conda-forge issue 81](https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/81) in [`upload_or_check_non_existence.py`](https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/master/recipe/upload_or_check_non_existence.py)

Each of the tracebacks from [CircleCI](https://circleci.com/gh/conda-forge/pvlib-python-feedstock/25?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), [Appveyor](https://ci.appveyor.com/project/conda-forge/pvlib-python-feedstock/build/1.0.32/job/ueb31ha7ia7bfx10), and [TravisCI](https://travis-ci.org/conda-forge/pvlib-python-feedstock/jobs/264567803) say the same thing, which is very similar to the error reported. The issue was fixed in October, so maybe retriggering the build will work now?

### TravisCI Job log
<details><pre>
Traceback (most recent call last):
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 133, in add_release
    aserver_api.release(username, package_name, version)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/__init__.py", line 377, in release
    self._check_response(res)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.NotFound: ("release version=u'0.5.0' does not exist", 404)
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/anaconda", line 6, in <module>
    sys.exit(binstar_client.scripts.cli.main())
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/scripts/cli.py", line 88, in main
    description=__doc__, version=version)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/scripts/cli.py", line 66, in binstar_main
    return args.main(args)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 242, in main
    args=args)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 180, in upload_package
    add_release(aserver_api, args, username, package_name, version, release_attrs)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 138, in add_release
    create_release(aserver_api, username, package_name, version, release_attrs)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 42, in create_release
    aserver_api.add_release(username, package_name, version, [], announce, release_attrs)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/__init__.py", line 417, in add_release
    self._check_response(res)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.Unauthorized: ('Authorization token is no longer valid', 401)
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/upload_or_check_non_existence", line 122, in <module>
    main()
  File "/Users/travis/miniconda3/bin/upload_or_check_non_existence", line 111, in main
    upload(cli, meta, owner, channel)
  File "/Users/travis/miniconda3/bin/upload_or_check_non_existence", line 62, in upload
    env=os.environ)
  File "/Users/travis/miniconda3/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['anaconda', '--quiet', '-t', 'binstar.token', 'upload', '/Users/travis/miniconda3/conda-bld/osx-64/pvlib-python-0.5.0-py36_0.tar.bz2', '--user=conda-forge', '--channel=main']' returned non-zero exit status 1.
The command "upload_or_check_non_existence ./recipe conda-forge --channel=main" exited with 1.
</pre></details>

### Appveyor CONSOLE
<details><pre>
upload_or_check_non_existence .\recipe conda-forge --channel=main
Using Anaconda API: https://api.anaconda.org
Using Anaconda API: https://api.anaconda.org
[ERROR] 
Traceback (most recent call last):
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\commands\upload.py", line 133, in add_release
    aserver_api.release(username, package_name, version)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\__init__.py", line 377, in release
    self._check_response(res)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.NotFound: ("release version=u'0.5.0' does not exist", 404)
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "C:\Miniconda36-x64\Scripts\anaconda-script.py", line 5, in <module>
    sys.exit(binstar_client.scripts.cli.main())
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\scripts\cli.py", line 88, in main
    description=__doc__, version=version)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\scripts\cli.py", line 66, in binstar_main
    return args.main(args)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\commands\upload.py", line 242, in main
    args=args)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\commands\upload.py", line 180, in upload_package
    add_release(aserver_api, args, username, package_name, version, release_attrs)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\commands\upload.py", line 138, in add_release
    create_release(aserver_api, username, package_name, version, release_attrs)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\commands\upload.py", line 42, in create_release
    aserver_api.add_release(username, package_name, version, [], announce, release_attrs)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\__init__.py", line 417, in add_release
    self._check_response(res)
  File "C:\Miniconda36-x64\lib\site-packages\binstar_client\__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.Unauthorized: ('Authorization token is no longer valid', 401)
Traceback (most recent call last):
  File "C:\Miniconda36-x64\Scripts\upload_or_check_non_existence.py", line 122, in <module>
    main()
  File "C:\Miniconda36-x64\Scripts\upload_or_check_non_existence.py", line 111, in main
    upload(cli, meta, owner, channel)
  File "C:\Miniconda36-x64\Scripts\upload_or_check_non_existence.py", line 62, in upload
    env=os.environ)
  File "C:\Miniconda36-x64\lib\subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['anaconda', '--quiet', '-t', 'binstar.token', 'upload', 'C:\\bld\\win-64\\pvlib-python-0.5.0-py36_0.tar.bz2', '--user=conda-forge', '--channel=main']' returned non-zero exit status 1.
Command exited with code 1
</pre></details>

### CircleCI
<details><pre>
+ upload_or_check_non_existence /recipe_root conda-forge --channel=main
Using Anaconda API: https://api.anaconda.org
Using Anaconda API: https://api.anaconda.org
[ERROR] 
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 133, in add_release
    aserver_api.release(username, package_name, version)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 377, in release
    self._check_response(res)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.NotFound: ("release version=u'0.5.0' does not exist", 404)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/bin/anaconda", line 6, in <module>
    sys.exit(binstar_client.scripts.cli.main())
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/scripts/cli.py", line 88, in main
    description=__doc__, version=version)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/scripts/cli.py", line 66, in binstar_main
    return args.main(args)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 242, in main
    args=args)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 180, in upload_package
    add_release(aserver_api, args, username, package_name, version, release_attrs)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 138, in add_release
    create_release(aserver_api, username, package_name, version, release_attrs)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/commands/upload.py", line 42, in create_release
    aserver_api.add_release(username, package_name, version, [], announce, release_attrs)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 417, in add_release
    self._check_response(res)
  File "/opt/conda/lib/python3.6/site-packages/binstar_client/__init__.py", line 216, in _check_response
    raise ErrCls(msg, res.status_code)
binstar_client.errors.Unauthorized: ('Authorization token is no longer valid', 401)
Traceback (most recent call last):
  File "/opt/conda/bin/upload_or_check_non_existence", line 122, in <module>
    main()
  File "/opt/conda/bin/upload_or_check_non_existence", line 111, in main
    upload(cli, meta, owner, channel)
  File "/opt/conda/bin/upload_or_check_non_existence", line 62, in upload
    env=os.environ)
  File "/opt/conda/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['anaconda', '--quiet', '-t', 'binstar.token', 'upload', '/feedstock_root/build_artefacts/linux-64/pvlib-python-0.5.0-py36_0.tar.bz2', '--user=conda-forge', '--channel=main']' returned non-zero exit status 1.
+ exit 1
Exited with code 1
</details></pre>

--- a/README.md
+++ b/README.md
@@ -66,6 +66,7 @@ To manage the continuous integration and simplify feedstock maintenance
 Using the ``conda-forge.yml`` within this repository, it is possible to re-render all of
 this feedstock's supporting files (e.g. the CI configuration files) with ``conda smithy rerender``.
 
+For more information please check the [conda-forge documentation](https://conda-forge.org/docs/).
 
 Terminology
 ===========

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -51,24 +51,24 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda update --yes --quiet conda
+    - cmd: conda.exe update --yes --quiet conda
 
     - cmd: set PYTHONUNBUFFERED=1
 
     # Add our channels.
-    - cmd: conda config --set show_channel_urls true
-    - cmd: conda config --remove channels defaults
-    - cmd: conda config --add channels defaults
-    - cmd: conda config --add channels conda-forge
+    - cmd: conda.exe config --set show_channel_urls true
+    - cmd: conda.exe config --remove channels defaults
+    - cmd: conda.exe config --add channels defaults
+    - cmd: conda.exe config --add channels conda-forge
 
     # Configure the VM.
-    - cmd: conda install -n root --quiet --yes conda-forge-build-setup
+    - cmd: conda.exe install -n root --quiet --yes conda-forge-build-setup
     - cmd: run_conda_forge_build_setup
 
 # Skip .NET project specific build phase.
 build: off
 
 test_script:
-    - conda build recipe --quiet
+    - conda.exe build recipe --quiet
 deploy_script:
     - cmd: upload_or_check_non_existence .\recipe conda-forge --channel=main

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,4 +1,4 @@
-{% set version = "0.5.0" %}
+{% set version = "0.5.1" %}
 
 package:
   name: pvlib-python
@@ -7,7 +7,7 @@ package:
 source:
   fn: pvlib-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pvlib/pvlib-{{ version }}.tar.gz
-  md5: 6c5dbf6760f84a237ac9b5c1c8f7c888
+  md5: f53a1790207ecaa39689aa97e40d6b75
 
 build:
   number: 0

